packages/lib/shim-init-node.ts

```diff
@@ -471,9 +472,10 @@ function shimInit(options: ShimInitOptions = null) {
 		}, options);
 	};
 
-	shim.fetchBlob = async function(url: any, options) {
+	shim.fetchBlob = async function(url: any, options: any, downloadController?: DownloadController) {
```

Comment:
The `options` parameter by convention should be the last one, so it means adding `downloadController` as an option